dynawo/sources/Models/CPP/Components/ModelLoadRestorativeWithLimits/DYNModelLoadRestorativeWithLimits.cpp

```diff
@@ -498,4 +499,18 @@ namespace DYN {
     addElement("running", Element::STRUCTURE, elements, mapElement);
     addSubElement("value", "running", Element::TERMINAL, name(), modelType(), elements, mapElement);
   }
+
+  void
+  ModelLoadRestorativeWithLimits::printInitValuesParameters(std::ofstream& fstream) {
+    fstream << std::setw(50) << std::left << "load_U0Pu" << std::right << " =" << std::setw(15) << DYN::double2String(u0Pu_) << std::endl;
```

Comment:
The parametersDynamic_ structure contains the names of the parameters and the method getSubModelParameterValue allows to access their values, maybe we should try to use it to avoid specific list that are hard to maintain

Reply:
Which specific list ?

Reply:
OK, PR will be redone